template.md

```diff
@@ -10,6 +10,20 @@
 - **Location**: CITY, COUNTRY
 - **Attendee information**: LINK TO REFLECTOR
 
+```mermaid
+---
+displayMode: compact
+---
+gantt
+    dateFormat  YYYY MM DD HH mm Z
+    tickInterval 3d
+    Proposal advancement deadline         : milestone, 1969/12/23 08:00 +0000,2h
+    Day 1 : , 1970/01/01 10:00 +0000,7h
+    Day 2 : , 1970/01/02 10:00 +0000,7h
+    Day 3 : , 1970/01/03 10:00 +0000,7h
```

Comment:
how come these don't have day1/day2/day3 like in the 09 one?

Reply:
this is the timezone. it varies from each meeting